tests/unit/clients/test_google_drive_client.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from tentaclio.clients import GoogleDriveFSClient
+
+
+class TestGoogleDriveFSClient:
+    @pytest.mark.parametrize(
+        ("url, drive, path_parts"),
+        (
+            ("gdrive://My Drive/", "root", ()),
+            ("gdrive://My Drive/path/to/dir/", "root", ("path", "to", "dir")),
+            ("gdrive://My Drive/path/to/dir/file.txt", "root", ("path", "to", "dir", "file.txt")),
```

Comment:
Would it catch a bad url like "gdrive://My Drive/path//to/dir/" ?
Not that it really matters, it is merely to try to put a comment on the review !

Reply:
that's a valid path, it would be on hands of the user to provide a correct one